Analysis/Tasks/PWGHF/HFCandidateCreator2Prong.cxx

```diff
@@ -159,6 +160,11 @@ struct HFCandidateCreator2ProngMC {
         particlesMC, array{candidate.index0_as<aod::BigTracksMC>(), candidate.index1_as<aod::BigTracksMC>()},
         421, array{+kPiPlus, -kKPlus}, true, &sign);
       result += sign * D0ToPiK * int8_t(indexRecD0 > -1);
+      // Jpsi → e+e-
+      auto indexRecJpsi = RecoDecay::getMatchedMCRec(
+        particlesMC, array{candidate.index0_as<aod::BigTracksMC>(), candidate.index1_as<aod::BigTracksMC>()},
```

Comment:
Use a variable to save the array of daughter track in the same way as it is in `HFCandidateCreator3Prong`: https://github.com/AliceO2Group/AliceO2/blob/dev/Analysis/Tasks/PWGHF/HFCandidateCreator3Prong.cxx#L159